src/ua/sumdu/j2se/Potapenko/tasks/ArrayTaskList.java

```diff
@@ -0,0 +1,76 @@
+package ua.sumdu.j2se.Potapenko.tasks;
+
+public class ArrayTaskList{
+    Task[] task = new Task[0];
```

Comment:
It's better to give it an appropriate name to make an accent on the fact it's an array. `tasks` will be better